src/gui/Src/Gui/HandlesView.cpp

```diff
@@ -317,11 +321,26 @@ void HandlesView::disableWindowSlot()
     enumWindows();
 }
 
+
 void HandlesView::followInDisasmSlot()
 {
     DbgCmdExec(QString("disasm %1").arg(mWindowsTable->mCurList->getCellContent(mWindowsTable->mCurList->getInitialSelection(), 0)));
 }
 
+void HandlesView::followInThreads()
+{
+    QString threadID = mWindowsTable->mCurList->getCellContent(mWindowsTable->mCurList->getInitialSelection(), 4);
+
+    if(Config()->getBool("Gui", "PidTidInHex"))
+    {
+        duint threadIDnum = strtoll(threadID.toUtf8().constData(), NULL, 0x10);
+        threadID = QString::number(threadIDnum);
+    }
+
+    DbgCmdExec(QString("showthreadid " + threadID));

```

Comment:
Doing `DbgCmdExec(QString("showthreadid %1").arg(threadID)` is a bit clearer to read. Also the casing of the variable should be `threadId` to keep things consistent

Reply:
Name of variable adjusted and used `.args(threadId)`